userdocs/profiles.dev/docs/author-docs/nested-profiles.md

```diff
@@ -0,0 +1,33 @@
+---
+sidebar_position: 5
+---
+
+# Nested profile artifacts
+
+Profiles can refer to other profiles in their list of artifacts.
+
+Simply use the `source` field:
+
+```yaml
+# ...
+spec:
+  # ...
+  artifacts:
+    - name: # the name of your artifact as you would like it to be known in the profile
+      profile:
+        source:
+          url: # required: fully qualified URL to the nested profile repository
+          branch: # optional: the repo branch if the profile is somewhere other than `main`
+          path: # optional: the relative path to a profile's directory within the repo
+          tag: # optional: the tag of the profile
+    # ...
+```
+
+:::danger
+Do not attempt to add a profile into its own list of artifacts.
```

Comment:
I think this should be a little more expanded. Saying that if a remote profile happens to include this same profile that will also error. Not only if the user adds it into its own list of artifacts.